.esmrc

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
This file is not bundled with the webpack build? Can we avoid external configuration files?